app/actions.py

```diff
@@ -953,9 +951,13 @@ def fileWrite(self):
         self.fileStat = os.stat(self.fullPath)
         self.setMessage('File saved')
       except Exception as e:
-        self.setMessage(
-            'Error writing file. The file did not save properly.',
-            color=3)
+        if self.isReadOnly:
+          self.setMessage("Permission error. Try modifing in sudo mode.",
+                          color=3)
+        else:
+          self.setMessage(
+              'Error writing file. The file did not save properly.',
+              color=3)
```

Comment:
Sorry, I think the color parameter is ignored right now. That could be fixed by using it in ci_edit/app/window.h around
```
      statusLine = tb.message[0]
```
By doing something like
```
      statusLine = tb.message[0]
      color = tb.message[1]
```

Reply:
Fixed colors for status line. Is the status line the only window that uses `setMessage()`? If not, I may need to override the function, since I'm giving it a default display color. Otherwise, every call to setMessage would need to specify the color and there will be no default. It will be more verbose, but maybe better for clarity?